### PR TITLE
Feature/GitHub actions

### PR DESCRIPTION
**Pull request by prokhorind**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.19
+      - name: Verify dependencies
+        run: go mod verify
+      - name: Build
+        run: go build -v ./...
+      - name: Run tests
+        run: go test -race -vet=off ./...
```
Comment (hanzei) [inline .github/workflows/ci.yml]: What is the deal with `-vet=off`?

**Review by hanzei — APPROVED** [review]
Thanks for the quick fix :+1:

File: .github/workflows/ci.yml (hunk 1 above)
Comment (prokhorind) [reply .github/workflows/ci.yml]: As part of building a test binary, go test runs go vet on the package
and its test source files to identify significant problems. If go vet
finds any problems, go test reports those and does not run the test
binary. https://pkg.go.dev/cmd/go/internal/test

I can remove this param. It was a part of example for CI config

Comment (hanzei) [reply .github/workflows/ci.yml]: :+1: for removing. Having some basic validation is helpful.

**Review by OstapMelnychuk — APPROVED** [review]

**Pull request merged**